prisma-fmt/src/get_dmmf.rs

```diff
@@ -0,0 +1,36 @@
+use serde::Deserialize;
+
+/// The implementation of the CLI getDmmf() utility and its JSON format.
+pub mod internal {
+    pub use dmmf::{dmmf_json_from_schema as get_dmmf, *};
+}
+
+#[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+struct GetDmmfParams {
+    prisma_schema: String,
+}
+
+pub(crate) fn get_dmmf(params: &str) -> String {
+    let params: GetDmmfParams = match serde_json::from_str(params) {
+        Ok(params) => params,
+        Err(serde_err) => {
+            panic!("Failed to deserialize GetDmmfParams: {}", serde_err,);
+        }
+    };
+
+    get_dmmf_impl(params)
+}
+
+fn get_dmmf_impl(params: GetDmmfParams) -> String {
+    // Note:
+    // - if the Prisma schema is not valid, this panics
+    // - we can't have a `psl::get_dmmf()` invocation here that mimics what we did in `get_config`, as that would create a circular dependency
+    //   between the `dmmf` and `psl` crates
```

Comment:
I don't understand this line.  In the end, we'll likely want to move the whole get_config and get_dmmf implementation to prisma_fmt, where they are now is an accident of history. I definitely wouldn't want to move get_dmmf to psl even if we could.

Reply:
Ok, it's safe to remove that comment